src/Nethermind/Nethermind.Core/Withdrawal.cs

```diff
@@ -28,7 +27,7 @@ public class Withdrawal
     /// <summary>
     /// Gets or sets the withdrawal amount in Wei.
     /// </summary>
-    public UInt256 Amount { get; set; }
+    public ulong Amount { get; set; }
```

Comment:
Can we name it explicitly `AmountGwei` in code, have `Amount` property evaluated from this property and just use Attributes to correctly map it to json? Would be lot easier and smaller change, right?

Reply:
Yeah, applied changes, take a look if you like them.
Not sure if `AmountGwei` or `AmountInGWei` will be better